resources/code_examples/analytics/order_beacon.yml

```
@@ -6,7 +6,7 @@
       revenue:       '1315.25',              // Grand Total. Includes Tax and Shipping. Does not include payment costs.
       shipping:      '5.45',                 // Total Shipping Cost. Does not include payment costs.
       tax:           '301.25',               // Total Tax.
-      paid_by:       'example_paid_by',      // [Optional] Payment method type, e.g. bank_transfer.
+      paid_by:       'example_paid_by',      // Payment method type, e.g. bank_transfer. Required for Technology categories.
       paid_by_descr: 'Example paid_by_descr' // [Optional] Payment method description, e.g. Bank transfer.
```

A few things that I was thinking about this section here.

First, we can consider simply dropping the `[Optional]` text only, without adding any further comments regarding the necessity of this field for shops that have products in categories under Technology.

Second, I'd also suggest we use an actual example for `paid_by` and `paid_by_descr` instead of `example_paid_by` and `Example paid_by_descr` respectively, so the example is closer to reality.

```suggestion
      paid_by:       'card',            // Payment method type, e.g. bank_transfer.
      paid_by_descr: 'Πληρωμή με κάρτα' // Payment method description, e.g. Bank transfer.
```